install1.sh

```diff
@@ -1,166 +1,31 @@
 #!/usr/bin/env bash
+touch /home/vagrant/.bashrc
 
-# setup directories
-mkdir -p /home/vagrant/dotfiles
-# mkdir -p /home/vagrant/.emacs.d
-mkdir -p /home/vagrant/downloads
-# If you are going to rsync your code don't forget to remove the appropriate lines
-# in the Vagrantfile
-# mkdir -p /home/vagrant/source
+# Setup directories
+mkdir -p /home/vagrant/source
 
-# CentOS/Fedora/RedHat
```

Comment:
I should put this commented stuff back since I don't want to ditch the possibility that we might want to use it in the future, we just want to default to Alpine for now.